index.js

```diff
@@ -80,6 +80,32 @@ const parseInput = async input => {
 	return input;
 };
 
+const killChecker = (input, options) => {
+	if (typeof options.verify !== 'undefined' && options.verify === false) {
+		return;
+	}
+
+	const seconds = options.verifyTimeout || 2;
+	let checks = seconds * 10;
+
+	function timeout() {
+		return new Promise(resolve => setTimeout(resolve));
+	}
+
+	async function checker() {
+		checks--;
+
+		const exists = await processExists(input);
+
+		if (exists && checks > 0) {
+			timeout();
+			await checker();
+		}
+	}
+
+	return checker();
+};
+
 const fkill = async (inputs, options = {}) => {
```

Comment:
Better to set defaults upfront.
```suggestion
const fkill = async (inputs, {verify = true, veryifyTimeout = 2} = {}) => {
```